install.py

```diff
@@ -1031,5 +1039,13 @@ def yes_or_no(value):
     return ('yes' if value else 'no')
 
 
+def search_for(path, filename):
```

Comment:
Auxiliary functions are listed +/- alphabetically in `install.py`'s source.  Can you pls move it among the others?